include/mgba/core/serialize.h

```diff
@@ -16,6 +16,7 @@ enum mStateExtdataTag {
 	EXTDATA_SAVEDATA = 2,
 	EXTDATA_CHEATS = 3,
 	EXTDATA_RTC = 4,
+	EXTDATA_HW_EXTENSIONS = 5,
```

Comment:
Extensions should probably have a larger ID, I'm trying to reserve low IDs for things that are standard on systems. Maybe 0x80.

Reply:
Maybe 0x1F? To make it consistent with the SAVESTATE_HW_EXTENSIONS define.
Since flags in mCoreLoadStateNamed and mCoreSaveStateNamed are declared as int, so I shouldn't exceed 32 bits.

Reply:
Fixed. Changed the enum to 0x80 and left the define as it was.